pkg/app/app_reconcile.go

```diff
@@ -190,12 +190,14 @@ func (a *App) resetLastDeployStartedAt() {
 func (a *App) reconcileInspect() error {
 	inspectResult := a.inspect().WithFriendlyYAMLStrings()
 
-	a.app.Status.Inspect = &v1alpha1.AppStatusInspect{
-		Stdout:    inspectResult.Stdout,
-		Stderr:    inspectResult.Stderr,
-		ExitCode:  inspectResult.ExitCode,
-		Error:     inspectResult.ErrorStr(),
-		UpdatedAt: metav1.NewTime(time.Now().UTC()),
+	if !inspectResult.IsEmpty() {
```

Comment:
i would recommend to have an else here as well, that clears inspect status if it inspectresult is empty.

Reply:
Sure, although I am curious to know if there are any chances of inspect status not being empty before this stage.

Reply:
it's more about enabling it, then disabling it.

Reply:
Done.